src/plots/cartesian/axis_defaults.js

```diff
@@ -161,6 +163,27 @@ function rangebreaksDefaults(itemIn, itemOut, containerOut) {
                 itemOut.bounds = itemOut.bounds.slice(0, 2);
```

Comment:
I don't think you have to do this `slice(0, 2)` - `coerce` for `info_array` already does this.
But if somehow I'm mistaken about that, you'd need to also shorten `bnds` so you don't lengthen `itemOut.bounds` again in the loops below.

Reply:
I don't see any place where `itemOut.bounds.length` is changed.

Reply:
`itemOut.bounds[i] = bnds[i] = q;`

Reply:
Good call. I revised the new logic in https://github.com/plotly/plotly.js/pull/4661/commits/550a3756da650fe3f3481fa13ce42092c39e5000.

Reply:
Again, you don't need to worry about the length of `bnds` at all - `coerce` handles that so `bnds` (and `itemOut.bounds`) is either `undefined` (since it has no default value) or it's an array of length 2 - or less, I just tested this, if you pass in a shorter array `coerce` isn't going to pad it since the inner values don't have defaults either, but that's OK, it'll just look like undefined for missing values and get filtered out in the type checks.

Can you please take all these length checks out? We put a lot of effort into making `coerce` do the right thing so the individual `supplyDefaults` functions could be simple and readable.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/4661/commits/84c2a6fa29a7b015a010ab5b52033a1318b9fece.